src/test/java/teammates/test/cases/browsertests/InstructorFeedbackSessionsPageUiTest.java

```diff
@@ -50,7 +51,7 @@ protected void prepareTestData() {
                 .withInstructions(new Text("Please fill in the new feedback session."))
                 .withSentOpenEmail(false)
                 .withSentPublishedEmail(false)
-                .withTimeZone(8.0)
+                .withTimeZone(TimeHelper.convertToZoneId(8))
```

Comment:
Just use something like `ZoneId.of("+08:00")`? `convertToZoneId` should be removed eventually.